### DISCUSSION TITLE
Ich möchte gerne einen Wert mehr per MQTT Publishen

### DISCUSSION THREAD

**Discussion by sonny1985** [matlen67/soyosource-powercontroller#10] [General]
Hallo :) Ich möchte mich hier auch beim Entwickler für dieses tolle Projekt bedanken. Es funktioniert einwandfrei und ich konnte keine Bugs feststellen ! Echt sensiationell 

Ich verwende es in Verbindung mit Iobroker, einem Trucki2Meanwell Gateway (Um Überschuss nicht zu verschenken sondern in den Lifepo Akku zu laden) Die Regelung des Ladegeräts und des Soyo Powercontrollers kommen sich nicht in die Quere :)

Ich hätte ein einziges Anliegen: Per MQTT wird ja ein Topic (Power) published.
Kann mir von euch jemand erklären wie ich ein weiteres publishen kann ? 
Ich würde gern den NULLSTATE per Mqtt wissen, um in einer Visualisierung zu sehen, ob der automatikbetrieb (Nulleinspeisung) aktiviert ist, oder nicht.
Momentan hole ich den Wert alle 5 sekunden beispielweise, aus der Json (/json)
Einfacher wäre es , wenn er direkt am MQTT bereitstehen würde. Verursacht dann natürlich auch weniger Netzwerklast.

Ich hab zumindest die html hinbekommen. (Ich lerne da noch und experimentiere viel)
Komme aber im Arduino (Ino datei) nicht weiter, blick hier bei den Variablen nicht durch.

Ich kann den Automatibetrieb zumindest per Befehl ein und ausschalten , was ich per Scripte in gewissen Umständen auch machen muss.
/checkbox?cbid=CBNULL&state=1
/checkbox?cbid=CBNULL&state=0

Nur wie bekomme ich das noch in den MQTT ? Eben, ob das nun aktiv ist oder nicht
Wäre um paar Infos dankbar :)
LG und angenehmes Wochenende.

**Comment by sonny1985** [comment]
Kurzer Nachtrag: Ich hab das nach gefühlten Ewigkeiten herumprobieren nun tatsächlich selbst hinbekommen .
Der iobroker Energiefluss adapter, kann das nun in Echtzeit darstellen :)

**Reply by matten67** [reply]
Hey moin, super das Du es hinbekommen hast.
